h2o-core/src/main/java/water/rapids/ast/prims/mungers/AstRBind.java

```diff
@@ -235,7 +235,6 @@ public void map(Chunk cs) {
       int idx = _chunkOffset + cs.cidx();
       Key ckey = Vec.chunkKey(_v._key, idx);
       if (_cmap != null) {
-        assert !cs.hasFloat() : "Input chunk (" + cs.getClass() + ") has float, but is expected to be categorical";
```

Comment:
hasFloat method needs to be fixed or replaced by something else; it's behavior can produce bugs elsewhere